### PR TITLE
Added licenses and dependencies for chunk5

### PR DESCRIPTION
**Pull request by tsg**
This was done by manually collecting license and URL information.

### CODE REVIEW

File: notices/rumbster-v1.1.0-NOTICE.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (jsvd) [inline notices/rumbster-v1.1.0-NOTICE.txt]: I think copyright could be copied from https://github.com/aesterline/rumbster#copyright

Comment (tsg) [reply notices/rumbster-v1.1.0-NOTICE.txt]: thanks, i added it

File: notices/org.apache.logging.log4j-NOTICE.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Apache Log4j
```
[the remark this answers is not in the document]
Comment (tsg) [reply notices/org.apache.logging.log4j-NOTICE.txt]: Yeah, this one I wasn't sure. This is what I had as input: 

```
org.apache.logging.log4j:log4j-api,2.8.2,UNKNOWN
org.apache.logging.log4j:log4j-api,2.9.1,Apache-2.0
org.apache.logging.log4j:log4j-core,2.6.2,UNKNOWN
org.apache.logging.log4j:log4j-core,2.9.1,Apache-2.0
org.apache.logging.log4j:log4j-slf4j-impl,2.8.2,UNKNOWN
org.apache.logging.log4j:log4j-slf4j-impl,2.8.2,UNKNOWN
org.apache.logging.log4j:log4j-slf4j-impl,2.9.1,Apache-2.0
```

Do we need files named like `org.apache.logging.log4j:log4j-api-2.8.2-NOTICE.txt` for each of the above? @andrewvc what does the automation expect?

**Comment by jsvd** [comment]
merged in https://github.com/elastic/logstash/pull/9886

**Pull request closed without merging**